src/gdriveApi.js

```diff
@@ -197,3 +198,35 @@ async function shareItem(id, shareData) {
 
   logger.log('verbose', 'gdrive - shareItem - done', id, shareData)
 }
+
+export async function getForeignCurrencyRates() {
+  const monthNames = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+  logger.log('verbose', 'gdrive - read spreadsheet')
+
+  // source google sheet for up-to-date currency rates
+  // https://docs.google.com/spreadsheets/d/1HfaU9jXZtCfdotEEfDO0d4EpDLyhmqdepaWgn_a6Lsw/edit#gid=1807750274
+  const fileId = '1HfaU9jXZtCfdotEEfDO0d4EpDLyhmqdepaWgn_a6Lsw'
+  const today = new Date()
+  const sheetName =  `${monthNames[today.getMonth() - 1]} ${today.getFullYear()}` // e.g. 'Nov 2022'
+
+  const {data:{values:rates}} = await sheets.spreadsheets.values.get({
+    spreadsheetId: fileId,
+    range: `${sheetName}!A2:B50`,
+  })
+
+  const formatToFixed = (num, decimals) => Number.parseFloat(num).toFixed(decimals)
+  // from:to ~ 1:value
+  const rate = {
+    EUR_EUR: 1,
+    CZK_CZK: 1,
+    HUF_HUF: 1,
+    EUR_CZK: formatToFixed(rates.find(([currency, rate]) => currency === 'CZK')[1], 4),
+    EUR_HUF: formatToFixed(rates.find(([currency, rate]) => currency === 'HUF')[1], 6),
```

Comment:
not sure why `formatToFixed` is used here. also, it returns string, while the other rates here (`1`) are numbers. while `parseFloat` can run on both numbers and strings, I think it would be reasonable to make the types of inputs and outputs consistent